src/app/shared/services/sentry/sentry.service.ts

```diff
@@ -12,16 +12,17 @@ export class SentryService {
     return Sentry.init({
       dsn: SENTRY_DSN,
       environment: this.envService.SENTRY_ENVIRONMENT,
+      integrations: [new BrowserTracing()],
       tracesSampleRate: 0.2,
     });
   }
 
   error(message: string) {
-    Sentry.captureMessage(message, Severity.Error);
```

Comment:
Is `Severity` no longer included in `@sentry/angular`

Reply:
It's deprecated.There is info in interface:
`Please use a SeverityLevel string instead of the Severity enum. Acceptable values are 'fatal', 'error', 'warning', 'log', 'info', and 'debug'.`

Reply:
where:
`export declare type SeverityLevel = 'fatal' | 'error' | 'warning' | 'log' | 'info' | 'debug';`

Reply:
Gotcha, thanks

Reply:
I had to update one more place with this SeverityLevel, because application wasn't building.